bin/copyQCDataToTmp.sh

```diff
@@ -357,44 +445,84 @@ done
-PRM_DARWIN_LOGS_DIR="/groups/${group}/${PRM_LFS}/logs/darwin/"
-
-mkdir -p "${PRM_DARWIN_LOGS_DIR}"
-
-
-
-readarray -t darwindata < <(find "${IMPORT_DIR}/" -maxdepth 1 -mindepth 1 -type f -name "*runinfo*" | sed -e "s|^${IMPORT_DIR}/||")
-
-if [[ "${#darwindata[@]}" -eq '0' ]]
-then
-	log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "no new darwin files present in ${IMPORT_DIR}"
-else
-	for darwinfile in "${darwindata[@]}"
-	do
-		log4Bash 'TRACE' "${LINENO}" "${FUNCNAME:-main}" '0' "Start proseccing ${darwinfile}"
-		runinfoFile=$(basename "${darwinfile}" .csv)
-		fileType=$(cut -d '_' -f1 <<< "${runinfoFile}")
-		fileDate=$(cut -d '_' -f3 <<< "${runinfoFile}")
-		tableFile="${fileType}_${fileDate}.csv"
-		runinfoCSV="${runinfoFile}.csv"
-		controlFileBase="${PRM_DARWIN_LOGS_DIR}"
-		DARWIN_JOB_CONTROLE_FILE_BASE="${controlFileBase}/darwin.${fileType}_${fileDate}.${SCRIPT_NAME}"
-
-		if [[ -e "${DARWIN_JOB_CONTROLE_FILE_BASE}.finished" ]]
-		then
-			log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "${DARWIN_JOB_CONTROLE_FILE_BASE}.finished present"
-			log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "${runinfoFile} data is already processed, but there is new data on dat05, check if previous rsync went okay"
-		else
-			log4Bash 'TRACE' "${LINENO}" "${FUNCNAME:-main}" '0' "no ${DARWIN_JOB_CONTROLE_FILE_BASE}.finished present, starting rsyncing ${tableFile} and ${runinfoCSV}"
-			copyDarwinQCData "${runinfoCSV}" "${tableFile}" "${fileType}" "${fileDate}" "${DARWIN_JOB_CONTROLE_FILE_BASE}"
-			log4Bash 'TRACE' "${LINENO}" "${FUNCNAME:-main}" '0' "${runinfoCSV} and ${tableFile} copied to tmp and moved to  ${IMPORT_DIR}/archive/"
-		fi
-	done
-fi
+	readarray -t darwindata < <(find "${IMPORT_DIR}/" -maxdepth 1 -mindepth 1 -type f -name "*runinfo*" | sed -e "s|^${IMPORT_DIR}/||")
+	
+	if [[ "${#darwindata[@]}" -eq '0' ]]
+	then
+		log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "no new darwin files present in ${IMPORT_DIR}"
+	else
+		for darwinfile in "${darwindata[@]}"
+		do
+			log4Bash 'TRACE' "${LINENO}" "${FUNCNAME:-main}" '0' "Start proseccing ${darwinfile}"
+			runinfoFile=$(basename "${darwinfile}" .csv)
+			fileType=$(cut -d '_' -f1 <<< "${runinfoFile}")
+			fileDate=$(cut -d '_' -f3 <<< "${runinfoFile}")
+			tableFile="${fileType}_${fileDate}.csv"
+			runinfoCSV="${runinfoFile}.csv"
+			controlFileBase="${DAT_ROOT_DIR}/logs/trendanalysis/"
+			DARWIN_JOB_CONTROLE_FILE_BASE="${controlFileBase}/${dat_dir}.${SCRIPT_NAME}.darwin"
+			DARWIN_JOB_CONTROLE_LINE_BASE="${fileType}-${fileDate}.${SCRIPT_NAME}"
+			if grep -Fxq "${DARWIN_JOB_CONTROLE_LINE_BASE}.finished" "${DARWIN_JOB_CONTROLE_FILE_BASE}"
+			then
+				log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "${DARWIN_JOB_CONTROLE_LINE_BASE}.finished present"
+				log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "${runinfoFile} data is already processed, but there is new data on dat05, check if previous rsync went okay"
```

Comment:
dat05 vervangen door ${dat_dir} denk ik

Reply:
de darwin data komt standaard binnen op dat05. ik verander het wel, is inderdaad mooier